src/index.ts

```diff
@@ -13,4 +13,6 @@ export * from './models/transactions'
 
 export * from './utils'
 
+export * from './errors'
```

Comment:
Do we want to be exporting errors?

Reply:
`catch` blocks or to check the error type

Reply:
I'm happy to take it out, but we can discuss this more in my PR for exporting things at the top level (which I'll put up by tomorrow)